tools/mmp/driver.cs

```diff
@@ -479,7 +479,7 @@ static void Main2 (string [] args)
 			}
 
 			if (!IsUnifiedMobile && tls_provider != null)
-				throw new MonoMacException (2011, true, "Selecting a TLS Provider is only supported in the Unified Mobile profile");
+				throw new MonoMacException (2011, true, "Selecting a TLS Provider is only supported in the Unified Modern (Mobile) profile");
```

Comment:
No, that's already done (in `mmp` too IIRC) and the linker code was adjusted accordingly. Right now it's a removal of the option (not the whole code).